### ISSUE TITLE
cross platform way to get process name?

### ISSUE THREAD

**Issue by jdalton** [simonepri/pidtree#5]
Hi!

With a given ID is there a cross-platform way to get the process name?

**Comment by soyuka** [comment]
not really, process would be extracted from the Command (on windows there is one, but it's almost the same). Anyway, I think that this is not really useful.
